tests/unit_tests/test_math.py

```diff
@@ -213,13 +213,24 @@ def test_normal_dist():
 
     assert ref_val == pytest.approx(test_val)
 
-    prn_seed = 1
-    a = 14.08
+    # make sigma = 1.0
     b = 1.0
-    ref_val = 16.436645416691427
-    test_val = openmc.lib.math.normal_variate(a, b, prn_seed)
 
-    assert ref_val == pytest.approx(test_val)
+    # import shapiro test 
+    from scipy.stats import shapiro
+    samples = []
+    num_samples = 10000
+    # sample some numbers
+    for i in range(num_samples):
+        # sample the normal distribution from openmc
+        samples.append(openmc.lib.math.normal_variate(a, b, prn_seed))
+        prn_seed = prn_seed + 1
```

Comment:
If we're confident this test will always pass, wouldn't it be ok to just use seeds that are not pre-determined? i.e., you'll get a different set of samples each time the test runs, but they should always be normally distributed